subsys/bluetooth/controller/ll_sw/ull_sync.c

```diff
@@ -205,17 +205,17 @@ uint8_t ll_sync_create_cancel(void **rx)
 
 	node_rx = (void *)scan->per_scan.node_rx_estab;
 	link_sync_estab = node_rx->hdr.link;
-	link_sync_lost = sync->node_rx_lost.link;
+	link_sync_lost = sync->node_rx_lost.hdr.link;
 
 	ll_rx_link_release(link_sync_lost);
 	ll_rx_link_release(link_sync_estab);
 	ll_rx_release(node_rx);
 
-	node_rx = (void *)&sync->node_rx_lost;
+	node_rx = (void *)&sync->node_rx_lost.hdr;
 	node_rx->hdr.type = NODE_RX_TYPE_SYNC;
 	node_rx->hdr.handle = 0xffff;
 	node_rx->hdr.rx_ftr.param = sync;
-	se = (void *)node_rx->pdu;
+	se = (void *)&sync->node_rx_lost.reason;
```

Comment:
I'm not sure I understand this. 

`se` is of type `struct node_rx_sync` and `sync->node_rx_lost.reason` is just a `uint8_t`. By doing this, you make `status == reason`, but if trying to access any other member of the `se` struct, you access illegal data.